projects/resettlement-passport-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/CaseNoteService.kt

```diff
@@ -17,11 +18,14 @@ class CaseNoteService(
     private val personRepository: PersonRepository,
     private val caseNoteTypeRepository: CaseNoteTypeRepository,
     private val assignmentService: AssignmentService,
+    private val alertRepository: AlertRepository,
+    private val personManagerRepository: PersonManagerRepository
 ) : AuditableService(auditedInteractionService) {
-
+    @Transactional
     fun createContact(crn: String, createContact: CreateContact) = audit(BusinessInteractionCode.ADD_CONTACT) {
         val person = personRepository.getByCrn(crn)
         val type = caseNoteTypeRepository.getCode(createContact.type.code)
+        val cm = lazy { personManagerRepository.getByCrn(crn) }
```

Comment:
Why does this need to be lazy?  We're always going to get the value anyway right?

Reply:
fair do's

Reply:
removed